test/etc/tasks/system/ExecTest.xml

```diff
@@ -206,12 +206,21 @@
         </exec>
     </target>
 
-    <target name="testEnvVars">
+    <target name="testEnvVar">
         <exec executable="echo" outputProperty="hello">
             <env key="HELLO" value="world"/>
             <arg value="$HELLO"/>
         </exec>
         <echo msg="${hello}"/>
     </target>
 
+    <target name="testMultipleEnvVars">
+        <exec executable="php" outputProperty="outputProperty">
+            <env key="HELLO" value="hello"/>
+            <env key="WORLD" value="world"/>
+            <arg value="-r"/>
+            <arg value="print '$HELLO ' . '$WORLD';"/>
```

Comment:
This seems a little bit strange to me - imagine one of the variables are used inside the code itself.

Reply:
Right, that would be a more conclusive assertion.  Updated.